scripts/build/.variables

```diff
@@ -34,8 +20,59 @@ if [ "${GOARCH}" = "arm" ] && [ -n "${GOARM}" ]; then
 fi
 
 if [ "${GOOS}" = "windows" ]; then
-	TARGET="${TARGET}.exe"
+    TARGET="${TARGET}.exe"
 fi
 export TARGET
 
+CGO_ENABLED=${CGO_ENABLED:-}
+if [ -z "$CGO_ENABLED" ]; then
```

Comment:
This looks odd; first we set the variable, and directly after, we check if it's empty?

Reply:
Yes was not necessary, removed.